plugins/modules/ipauser.py

```diff
@@ -19,6 +19,12 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+#
+# TEMP COMMIT: This will enable the test for modules:
+#     automember, cert, group, hbacrule, host, hostgroup
```

Comment:
The executed tests are different per distribution.

The test for the module cert is not executed for all distributions in run #20230929.2:

Not executed:
- Fedora_Latest
- Galaxy_Fedora_Latest
- CentOS_8_Stream

Executed:
- CentOS_9_Stream
- Fedora_Rawhide

There seems to be another issue.

Reply:
The problem was that the random seed used to randomly split tests was not the same for every test group in the same build. This was also affecting other pipelines, including Nightly and Weekly tests.

This issue has been fixed.